tests/test_training_data.py

```diff
@@ -103,3 +105,17 @@ def test_get_subset(training_data_factory, project_factory, td_factory_args):
         json={"subset": td_subset},
     )
     assert td.get_subset(project=project) == td_subset
+
+
+@responses.activate
+def test_set_subset(training_data_factory, project_factory):
+    project = project_factory(id="n07e45y", name="bananarama")
+    td: TrainingData = training_data_factory(project=project, subset=SubsetEnum.TRAINING)
+
+    responses.add(
+        responses.GET,
+        f"https://test.test/projects/{project.id}/data/{td.id}/subset",
+        status=200,
+        json={"subset": SubsetEnum.VALIDATION},
+    )
+    assert td.get_subset(project=project) == SubsetEnum.VALIDATION
```

Comment:
Feels like I already wrote that comment before:
we're in `test_set_subset`, why are we testing `td.get_subset` ?